### ISSUE TITLE
JVM does not stop after end of all tests

### ISSUE THREAD

**Issue by vmi** [vmi/selenese-runner-java#244]
This is causes the following comment problem:
https://github.com/vmi/selenese-runner-java/issues/148#issuecomment-305835150

**Comment by vmi** [comment]
Hi, @jschwartzenberg

I fixed the problem of hanging up with `--no-exit` option and released new version 3.6.0.
However,  you need to add the following settings when using _exec-maven-plugin_.
```
<configuration>
  <cleanupDaemonThreads>false</cleanupDaemonThreads>
  ...
</configuration>
```
Please test it.

**Comment by jschwartzenberg** [comment]
I just tested it and I can confirm it works properly now. Thanks!
